### ISSUE TITLE
Build fails on AIX due to missing timeradd/timersub/timercmp functions

### ISSUE THREAD

**Issue by kadler** [psycopg/psycopg2#1055]
When attempting to build psycopg2 we get the following errors during the link step:

```
ld: 0711-317 ERROR: Undefined symbol: .timeradd
ld: 0711-317 ERROR: Undefined symbol: .timersub
```

AIX does not supply these BSD functions. It seems that there are already definitions for them in [solaris_support.c](https://github.com/psycopg/psycopg2/blob/master/psycopg/solaris_support.c), but that only gets built on Solaris-derived systems. Likewise there are functions defined in [win32_support.c](https://github.com/psycopg/psycopg2/blob/master/psycopg/win32_support.c#L65). 

Considering the code is the same for Solaris and AIX except for precompiler definitions, would it make sense to reuse solaris_support.{c.h} for AIX (maybe with a file rename) or would it be better to copy the code to new files aix_support.{c,h}?

**Comment by dvarrazzo** [comment]
Please provide a patch, thank you.

**Comment by dvarrazzo** [comment]
Yes sure: happy to receive a MR, and using a pattern similar to the solaris support files would work perfectly well :)

**Comment by dvarrazzo** [comment]
Fixed by #1061.
